### DISCUSSION TITLE
File Upload

### DISCUSSION THREAD

**Discussion by croche2574** [neonwatty/meme-search#49] [Ideas]
Add an option to upload images to the storage. I run this on my NAS and adding files would be easier with a web form.

**Comment by neonwatty** [comment]
Great feature suggestion.

**Comment by neonwatty** [comment]
@croche2574  - drag-and-drop uploads functionality now live in version 2.0.1!

![uploads-trimmed](https://github.com/user-attachments/assets/cfb80c8f-789e-45a8-aeea-abd124769bb8)
